lodash/lodash.d.ts

```diff
@@ -384,7 +384,15 @@ declare module _ {
         unshift(...items: T[]): LoDashImplicitArrayWrapper<T>;
     }
 
-    interface LoDashExplicitArrayWrapper<T> extends LoDashExplicitWrapperBase<T[], LoDashExplicitArrayWrapper<T>> { }
+    interface LoDashExplicitArrayWrapper<T> extends LoDashExplicitWrapperBase<T[], LoDashExplicitArrayWrapper<T>> {
+        pop(): LoDashExplicitObjectWrapper<T>;
```

Comment:
This is somewhat unfortunate, but I'm not sure how to overload this to select between other types of wrappers (e.g., `pop`ing on a 2D array wrapper should yield a 1D array wrapper, but this will yield an object wrapper).